.github/workflows/publish.yml

```
@@ -0,0 +1,34 @@
+name: publish
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - '*/PKGBUILD'
```

```suggestion
      - '**/PKGBUILD'
```

I think we need the double `**`?